bsc-plugin/src/lib/rooibos/TestGroup.ts

```diff
@@ -60,19 +61,20 @@ export class TestGroup extends TestBlock {
                         let assertRegex = /(?:fail|assert(?:[a-z0-9]*)|expect(?:[a-z0-9]*)|stubCall)/i;
                         if (dge && assertRegex.test(dge.name.text)) {
                             if (dge.name.text === 'stubCall') {
-                                this.modifyModernRooibosExpectCallExpression(callExpression);
+                                this.modifyModernRooibosExpectCallExpression(callExpression, editor);
                                 return expressionStatement;
 
                             } else {
 
                                 if (dge.name.text === 'expectCalled' || dge.name.text === 'expectNotCalled') {
-                                    this.modifyModernRooibosExpectCallExpression(callExpression);
+                                    this.modifyModernRooibosExpectCallExpression(callExpression, editor);
                                 }
-                                return new RawCodeStatement(`
-                                m.currentAssertLineNumber = ${callExpression.range.start.line}
-                                ${callExpression.transpile(transpileState).join('')}
-                                ${noEarlyExit ? '' : 'if m.currentResult.isFail then return invalid'}
-                                `, this.file, callExpression.range);
+                                //TODO change this to editor.setProperty(parentObj, parentKey, new SourceNode()) once bsc supports it
```

Comment:
This is a future enhancement...but the `overrideAstTranspile` function I added in `utils` will work fine until then.